api/src/utils/oracle-schema.ts

```diff
@@ -0,0 +1,9 @@
+import { Knex } from 'knex';
+
+export async function oracleForceAlterColumn(knex: Knex, table: string, column: string, type: string): Promise<void> {
+	await knex.raw(`ALTER TABLE "${table}" ADD "${column}__temp" ${type}`);
```

Comment:
Lets use Knex's identifier syntax here to make sure we have some sql injection safety here (even though it's only us that uses this now):

```suggestion
	await knex.raw(`ALTER TABLE ?? ADD ?? ?`, [table, `${column}__temp`, type]);
```